### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by YevgeniySavchuk**

### CODE REVIEW

File: src/figuresClasses.ts [hunk 1]
```diff
@@ -1,19 +1,76 @@
 export interface Figure {
-
+  shape: string;
+  color: 'red' | 'green' | 'blue';
```
Comment (vkryvytskyy) [inline src/figuresClasses.ts:3]: create type Color

File: src/figuresClasses.ts [hunk 2]
```diff
@@ -1,19 +1,76 @@
 export interface Figure {
-
+  shape: string;
```
Comment (vkryvytskyy) [inline src/figuresClasses.ts:2]: create type Shape

**Review by vkryvytskyy — CHANGES_REQUESTED** [review]

File: src/figuresClasses.ts [hunk 3]
```diff
@@ -1,19 +1,79 @@
...
+      a >= (b + c)
+      || b >= (a + c)
+      || c >= (a + b)
+    ) {
+      throw new Error('There is now triangle with such dimentions.');
+    }
+  }
 
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
```
Comment (marina-tilniak) [inline src/figuresClasses.ts:33]: what is s? Use meaningful names for variables

**Review by marina-tilniak — APPROVED** [review]
Good 👍  Fix comment ⏬

**Pull request closed without merging**